### PR TITLE
Jubeclean

### PR DESCRIPTION
**Pull request by sdiazpier**
This PR addresses issue #7

### CODE REVIEW

File: bin/l2l-funall.py [hunk 1]
```diff
@@ -45,7 +46,8 @@ def main():
         (CrossEntropyOptimizer,
          lambda: CrossEntropyParameters(pop_size=50, rho=0.2, smoothing=0.0, temp_decay=0,
                                         n_iteration=n_iterations,
-                                        distribution=NoisyGaussian(noise_decay=0.95, noise_bias=0.05))),
+                                        distribution=NoisyGaussian(noise_decay=0.95),
```
Comment (alperyeg) [inline bin/l2l-funall.py:49]: Did you remove `noise_bias`?

**Review by alperyeg — COMMENTED** [review]
Just a few questions, rest seems fine.

File: bin/l2l-funall.py (hunk 1 above)
Comment (sdiazpier) [reply bin/l2l-funall.py:49]: No this parameter is still there, in the previous to last line

File: bin/l2l-fun-ga.py [hunk 2]
```diff
@@ -29,7 +29,7 @@ def main():
         )
     paths = Paths(name, dict(run_no='test'), root_dir_path=root_dir_path)
 
-    with open("bin/logging.yaml") as f:
+    with open("logging.yaml") as f:
```
[the remark this answers is not in the document]
Comment (sdiazpier) [reply bin/l2l-fun-ga.py:32]: We probably don't need the yaml file... we should add an issue to take a look at the logger entirely :(

[the remark this answers is not in the document]
Comment (alperyeg) [reply bin/l2l-fun-ga.py:32]: I will make an issue for it

**Pull request merged**